deno/src/core/client.ts

```diff
@@ -235,11 +241,10 @@ class ApiClient<R extends RawApi> {
                 })
             } catch (err) {
                 let msg = `Network request for '${method}' failed!`
-                if (isTelegramError(err)) {
+                if (isTelegramError(err))
                     msg += ` (${err.status}: ${err.statusText})`
-                } else if (this.options.sensitiveLogs && err instanceof Error) {
+                if (this.options.sensitiveLogs && err instanceof Error)
                     msg += ` ${err.message}`
-                }
```

Comment:
Personally I wouldn't use if without {} but thats probably personal taste.

Reply:
I do it sometimes … only if the code is still readable enough, however subjective that may be